docs/user_documentation/guide-quick-upload.md

```diff
@@ -0,0 +1,47 @@
+Quick import
+------------
+
+The Quick data import allows you to easily get your data into MOLGENIS, without the need of understanding the EMX format.
+There are some assumptions we make with regards file types and file contents.
+
+1. The supported file types are _CSV_, _XLS_, _XLSX_, and a _ZIP_ file containing _CSV_ files
+2. A file consists of a at least two lines, the first is the header, the rest is data
+3. We always expect the first line to be the header 
+
+We do not accept empty files or files with only a header.
```

Comment:
Maybe remove the we part 

"Empty files can not be imported"
"Files containing only a header can not be uploaded"